tls/s2n_send.c

```diff
@@ -145,6 +145,8 @@ ssize_t s2n_sendv_with_offset_impl(struct s2n_connection *conn, const struct iov
         }
         conn->last_write_elapsed = elapsed;
     }
+    
+    POSIX_GUARD(s2n_stuffer_rewrite(&conn->out));
```

Comment:
Is this necessary? If it is necessary, than wouldn't you have an issue with the s2n_flush call that you added to the post handshake messages / new session ticket?

Reply:
I'll remove this. It's redundant with the flush [here](https://github.com/aws/s2n-tls/blob/da03bc958f63177e3cfa90ed82d59803f3f59a18/tls/s2n_send.c#L99). 

Can you expand on the issue you see with the post handshake and new session ticket flushes? My thought is that if they want to use the out stuffer and flush the bytes on it they should make sure that it has been flushed first.